app/uk/gov/hmrc/helptosavefrontend/config/FrontendWiring.scala

```diff
@@ -16,30 +16,48 @@
 
 package uk.gov.hmrc.helptosavefrontend.config
 
-import com.google.inject.{Inject, Singleton}
+import com.google.inject.{ImplementedBy, Inject, Singleton}
 import play.api.http.HttpVerbs.{GET ⇒ GET_VERB, POST ⇒ POST_VERB}
 import play.api.libs.json.Writes
+import play.api.libs.ws.WSProxyServer
 import uk.gov.hmrc.auth.core.PlayAuthConnector
 import uk.gov.hmrc.play.audit.http.HttpAuditing
-import uk.gov.hmrc.play.audit.http.config.LoadAuditingConfig
+import uk.gov.hmrc.play.audit.http.config.{AuditingConfig, LoadAuditingConfig}
 import uk.gov.hmrc.play.audit.http.connector.{AuditConnector ⇒ Auditing}
 import uk.gov.hmrc.play.config.{AppName, RunMode, ServicesConfig}
+import uk.gov.hmrc.play.http.hooks.HttpHook
 import uk.gov.hmrc.play.http.ws._
 import uk.gov.hmrc.play.http.{HeaderCarrier, HttpResponse, ws}
 
 import scala.concurrent.Future
 
 object FrontendAuditConnector extends Auditing with AppName {
-  override lazy val auditingConfig = LoadAuditingConfig("auditing")
+  override lazy val auditingConfig: AuditingConfig = LoadAuditingConfig("auditing")
 }
 
-trait WSHttpExtension extends WSGet with WSPost {
+@ImplementedBy(classOf[WSHttpExtension])
+trait WSHttp extends WSGet with WSPut with WSPost with WSDelete {
+
+  def get(url: String)(implicit rhc: HeaderCarrier): Future[HttpResponse]
+
+  def post[A](url:     String,
+              body:    A,
+              headers: Seq[(String, String)] = Seq.empty[(String, String)]
+  )(implicit rds: Writes[A], hc: HeaderCarrier): Future[HttpResponse]
+
+}
+
+@Singleton
+class WSHttpExtension extends WSHttp {
+
+  override val hooks: Seq[HttpHook] = NoneRequired
 
   /**
    * Returns a [[Future[HttpResponse]] without throwing exceptions if the status us not `2xx`. Needed
    * to replace [[GET]] method provided by the hmrc library which will throw exceptions in such cases.
    */
   def get(url: String)(implicit rhc: HeaderCarrier): Future[HttpResponse] = withTracing(GET_VERB, url) {
+    // Future.successful(1)
```

Comment:
can we remove this comment ?

Reply:
whoops, yeah I was using that to test the warts library